sdk/mixedreality/mixedreality-authentication/src/util/authenticationEndpoint.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export function constructAuthenticationEndpointFromDomain(accountDomain: string): string {
```

Comment:
I believe you'll need at least an `@internal` doc comment to prevent this from showing up in documentation as well. The current doc generation tools don't look at just what's re-exported by index.ts.

Reply:
Yea. That thing appears to pick up tons of stuff. I had to add `@ignore` to remove it. See my question above about possibly adding both.